index.rst

```diff
@@ -332,6 +332,13 @@ Welcome to PyTorch Tutorials
    :link: intermediate/rpc_param_server_tutorial.html
    :tags: Parallel-and-Distributed-Training
 
+.. customcarditem::
```

Comment:
We also need to add an item to the left navigation bar, which is at the end of this file (search for `:caption: Parallel and Distributed Training`).